docs/using.rst

```diff
@@ -376,10 +376,10 @@ The ``_p_serial`` attribute is not affected by calling setstate.
 
 .. doctest::
 
-   >>> p._p_serial = "00000012"
+   >>> p._p_serial = b"00000012"
    >>> p.__setstate__(p.__getstate__())
    >>> p._p_serial
-   '00000012'
+   b'00000012'
```

Comment:
Wow are these doctests never executed on Travis CI?

Reply:
The doctests are currently using sphinx, which makes it somewhat more complicated to write cross-version tests (no normalizers).

Reply:
I don't think so but until this PR they where run on Python 2.7.

Reply:
Ah, right.  Never mind then.

(But maybe we want to add a build job to run one sphinx-build doctest for the specific Python 3 version we're specifying in tox.ini).